.github/workflows/lint.yml

```
@@ -10,4 +10,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
+        with:
+        python-version: "3.9.7"
       - uses: pre-commit/action@v2.0.0
```

```suggestion
        with:
          python-version: "3.9.7"
      - uses: pre-commit/action@v2.0.3
```

Pay attention to the indentation.

Also change pre-commit/action to v2.0.3, following suit to the community template: https://github.com/OCA/oca-addons-repo-template/blob/2ed3a162cb4c78658dd6405abffe6857e8906178/src/.github/workflows/pre-commit.yml.jinja#L20